File: whospitthatAPI/src/main/java/com/quizz/whospitthatapi/entity/Question.java
package com.quizz.whospitthatapi.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.lang.reflect.Array;
import java.util.List;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Data
@Builder
@Table(name="questions")
public class Question {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private String questionText;
    private List<String> answers;
    private String goodAnswer;
    private String mediaUrl;
    private QuestionType questionType;
}

File: whospitthatFront/src/app/models/question.ts
import {QuestionType} from './questionType';

export class Question{
  idQuestion!: string;
  questionText!: string;
  answers!: string[];
  goodAnswer!: string;
  mediaUrl?: string;
  questionType!: QuestionType;

  constructor(idQuestion: string,questionText: string, answers:string[], goodAnswer: string, mediaUrl: string, questionType: QuestionType) {
    this.idQuestion = idQuestion;
    this.questionText = questionText;
    this.answers = answers;
    this.goodAnswer = goodAnswer;
    this.mediaUrl = mediaUrl;
    this.questionType = questionType;
  }

}
